### ISSUE TITLE
Could not get the files after running training code

### ISSUE THREAD

**Issue by boba1234** [orobix/retina-unet#77]
After running the code python run_training.py, i could not get the following files 
model architecture (json)
picture of the model structure (png)
a copy of the configuration file
model weights at last epoch (HDF5)
model weights at best epoch, i.e. minimum validation loss (HDF5)
sample of the training patches and their corresponding ground truth (png)

Does anyone know the reason why?

**Comment by boba1234** [comment]
It just stops at
2. Run the training on GPU with nohup 
without running any epoches

**Comment by tphankr** [comment]
> It just stops at
> 2. Run the training on GPU with nohup
> without running any epoches

@boba1234 
It means, we had like this, it is okies:
![image](https://user-images.githubusercontent.com/37217354/128612017-cfa03bcd-dcbc-402f-88cd-da03fcadf865.png)
Thank you.

**Comment by thetravelingtechdude** [comment]
Has anybody found a fix for this issue?
